test/Change.lean

```diff
@@ -0,0 +1,20 @@
+import Mathlib.Tactic.Change
+import Std.Tactic.GuardMsgs
+
+set_option pp.unicode.fun true
+
```

Comment:
```suggestion
```

Reply:
I thought that this was useful since otherwise the `fun` arrow gives incompatible errors for `guard_msgs`.  Am I wrong?

I was in this situation, where either the error was on my local machine or on CI until Kyle mentioned this option.

Reply:
Oh, maybe CI works differently? I can confirm that locally the `fun` arrows print as `↦` for me (and `#guard_msgs` sees them that way too). I suppose it doesn't hurt to ensure it. :)

Reply:
If you are curious, try looking at the tests for ExtractGoal: I had to use it there and there I tested it.  Here I just placed it based on memory.

Reply:
https://github.com/leanprover-community/mathlib4/pull/4595#issuecomment-1662171855
and below.

Reply:
Ah, I see! Thanks for the reference!